pydantic/warnings.py

```diff
@@ -6,8 +6,11 @@
 
 __all__ = (
     'PydanticDeprecatedSince20',
-    'PydanticDeprecationWarning',
```

Comment:
We should probably still export this, might be used in user code, and we export in `__init__.py`?

Reply:
It is still exported, just reordered.